src/plugins/kibana_usage_collection/server/collectors/management/README.md

```diff
@@ -5,11 +5,24 @@ All user changed UI Settings are automatically collected.
 
 After adding a new setting you will be required to do the following steps:
 
-1. Update the [schema](./schema.ts) to include the setting name and schema type.
+1. Update the schema to include the setting name, schema type and a short description.
 ```
-export const stackManagementSchema: MakeSchemaFrom<UsageStats> = {
-  'MY_UI_SETTING': { type: 'keyword' },
-}
+export function registerManagementUsageCollector(
+  usageCollection: UsageCollectionSetup,
+  getUiSettingsClient: () => IUiSettingsClient | undefined
+) {
+  const collector = usageCollection.makeUsageCollector<UsageStats | undefined>({
+    type: 'stack_management',
+    isReady: () => typeof getUiSettingsClient() !== 'undefined',
+    fetch: createCollectorFetch(getUiSettingsClient),
+    schema: {
```

Comment:
The `schema`'s declared in-line and I updated the example to reflect the change

Reply:
I've reverted my changes to the README since we are actually importing the schema now with the slight change to how the schema's `type` is declared.